### ISSUE TITLE
[SUGGESTION] Territect Discord server

### ISSUE THREAD

**Issue by z4dg** [Rebmiami/Territect#26]
**Is your feature request related to a problem? Please describe.**
Some people dont have a github for suggestions or bug reports . Also i think territect users would like to discuss Territect-related stuff in a separate server and not in tpt

**Describe the solution you'd like**
Pretty simple discord server for people to chat / share presets / help others etc

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by Rebmiami** [comment]
Good idea, I'll create one and add a link to it in the next version/on the help page.
